Adds new integration [alryaz/hass-tns-energo]

alryaz opened the pull request:
<!--
DO NOT REQUEST REVIEWS, THAT IS JUST RUDE, IF YOU DO THE PULL REQUEST WILL BE CLOSED!
Make sure to check out the guide here: https://hacs.xyz/docs/publish/start
And consider adding a GitHub Action workflow to your repository: https://hacs.xyz/docs/publish/action
-->

This is a submission of a functional adapter for one of the larger <!-- not a typo --> energy vendors in Russia, namely **TNS Energo**.

The addon supports a lot of personal cabinets (12, to be precise), and allows indications submission and account balance information polling.

It was developed in collaboration with multiple members of the Russian Home Assistant community.

I would like to have it added for the ease of support as well as to ensure most limitations are eased as they appear, and bugs fixed when encountered. Community contributions and outreach is crucial to ensure further development.

jbouwh reviewed (changes requested):
There is still a TODO in README.md
https://github.com/alryaz/hass-tns-energo#readme
Please complete documentation.
Is there a short English description available?

alryaz commented:
Alright, will deal with TODO's today.

To be honest, this addon is almost exclusive to Russian users; however, I think, a little English description wouldn't hurt.
(In short, this is an energy website adapter)

alryaz commented:
Removed that `@TODO@` entry, added a few screenshots and an English preamable.

I'd love to add more English README's, but since most of my components are oriented around adapting personal cabinets to HA, and there are so many similarities between them (service calls, attributes, etc.), I'm working on a README generator based on their traits. Not yet done, but README will be blooming with data when it's done.
